qadi_simple.py

```
@@ -0,0 +1,342 @@
+    """QADI orchestrator with simplified Phase 1."""
+    
+    def __init__(self, temperature_override: Optional[float] = None) -> None:
+        super().__init__(temperature_override)
+        # Use custom prompts
+        self.prompts = SimplerQADIPrompts()
+
+
+async def run_qadi_analysis(
+    user_input: str, 
+    temperature: Optional[float] = None, 
+    verbose: bool = False,
```

Ruff (0.12.2)

Boolean-typed positional argument in function definition

(FBT001)